packages/opds-web-client/src/OPDSDataAdapter.ts

```diff
@@ -280,6 +281,24 @@ export function feedToCollection(feed: OPDSFeed, feedUrl: string): CollectionDat
     if (shelfLink) {
       shelfUrl = shelfLink.href;
     }
+
+    let aboutLabels = {
+      "about": "About",
+      "terms-of-service": "Terms of Service",
+      "privacy-policy": "Privacy Policy",
+      "copyright": "Copyright"
+    };
```

Comment:
I looked through some other OPDS feeds and none of them had links like this, so maybe it shouldn't even be here - maybe this should have a configurable footer and circulation-patron-web should handle the links.